lib/omnibus/software.rb

```diff
@@ -1080,6 +1096,11 @@ def build_me(build_wrappers = [])
             "Building because `#{project.culprit.name}' dirtied the cache"
           end
           execute_build(build_wrappers)
+        elsif always_build
+          log.info(log_key) do
+            "Building because always_build is true"
+          end
+          execute_build(build_wrappers)
```

Comment:
You're right, it doesn't really matter because of the changes to the build order, but  it's better.